mailbag/__init__.py

```diff
@@ -16,6 +16,25 @@
 mailbagit_args = bagit_parser.add_argument_group("Mailbag")
 # add mailbag-specific args here
 mailbagit_args.add_argument("--foo", help="The foo argument, you know, that one")
+mailbagit_args.add_argument("--i","--input_type", help="input type MBOX/PST/IMAP/EML/PDF/MSG",nargs=1)
+mailbagit_args.add_argument("--input_path", help="input path MBOX/PST/IMAP/EML/PDF",nargs=1)
```

Comment:
I think its common practice to have a full arg with two dashes (`--input`) and an abbreviated arg with just a single letter with one dash (`-i`). This is typically done with argparse with the first arg being the abbreviated one. We should do this for all the args. I also don't think we need an `--input_path` arg, as we can use the required bagit directory arg for this. abd try to update the description. Having just an `--input` and `--output` type args should be more user-friendly.